pkg/apis/machineremediation/register.go

```diff
@@ -0,0 +1,4 @@
+package machineremediation
+
+// GroupName contains the name of the API group
+const GroupName = "machineremediation.kubevirt.io"
```

Comment:
I think we want this to be metal3 not KubeVirt.

```suggestion
const GroupName = "machineremediation.metal3.io"
```